package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "metal-state-validators",
+  "version": "1.0.0",
+  "description": "Utility class with helpers for creating validators for Metal.js state",
+  "main": "lib/type.js",
+  "scripts": {
+    "compile": "babel --presets es2015 -d lib/ src/",
+    "prepublish": "npm run compile",
+    "test": "mocha --compilers js:babel-core/register ./test/**/*.js"
```

Comment:
Not an issue, it just enforces the standard workflow. And in the end this might end up using metal.js core at least, as I mentioned now in another comment below :)